Singular/LIB/ncHilb.lib

```diff
@@ -1,50 +1,198 @@
 //////////////////////////////////////////////////////////////////////////////
 version="version nc_hilb.lib 4.1.1.0 Dec_2017 "; // $Id$
-category="Noncommutative algebra";
+category="Noncommutative";
 info="
-LIBRARY:  ncHilb.lib: A library for computing graded and multi-graded Hilbert
-                      series of non-commutative algebras. It also computes
-                      the truncated Hilbert series of the algebras whose Hilbert
-                      series are possibly not rational or unknown.
+LIBRARY:  fpahilb.lib: Computation of graded and multi-graded Hilbert series of non-commutative algebras (Letterplace). 
 
 AUTHOR:   Sharwan K. Tiwari   shrawant@gmail.com
           Roberto La Scala
-
+		  Viktor Levandovskyy (adaptation to the new Letterplace release)
+		  
 REFERENCES:
-          La Scala R.: Monomial right ideals and the Hilbert series of
-          non-commutative modules, J. of Symb. Comp. (2016).
+          La Scala R.: Monomial right ideals and the Hilbert series of non-commutative modules, 
+		  Journal of Symbolic Computation (2016).
 
-          La Scala R., Tiwari Sharwan K.: Multigraded Hilbert Series of
-          noncommutative modules, https://arxiv.org/abs/1705.01083.
+          La Scala R., Tiwari Sharwan K.: Multigraded Hilbert Series of noncommutative modules, https://arxiv.org/abs/1705.01083.
 
+KEYWORDS: finitely presented algebra; infinitely presented algebra; graded Hilbert series; multi-graded Hilbert series;
+		  
 PROCEDURES:
-          nchilb(L,d,#); Hilbert series of a non-commutative algebra
-          rcolon(L, w, d); Right colon ideal of a two-sided monomial ideal
-                           with respect to a monomial w
+          fpahilb(L,d,#); Hilbert series of a non-commutative algebra
+          rcolon(I, w, d); Right colon ideal of a two-sided monomial ideal with respect to a monomial w
 
 ";
 
 LIB "freegb.lib";
+// under construction by VL
+/*
```

Comment:
cannot load this lib: this comment ends in line 104, making the whole lib unusable

Reply:
ok, I am working on this lib anyway and it will be changed again. Is it possible to ignore the changes in this lib for the moment?